Makefile

```diff
@@ -16,7 +16,7 @@ GOPATH          := $(firstword $(subst :, ,$(shell $(GO) env GOPATH)))
 PROMU           ?= $(GOPATH)/bin/promu
 GOLINTER        ?= $(GOPATH)/bin/gometalinter
 #aligncheck and gosimple took unfortunately too long at travisCI
-GOLINTER_OPT    ?= --vendor --deadline 6m --cyclo-over=20 --disable=aligncheck --disable=gosimple
```

Comment:
Just for my curiosity, what was the issue with `gotype`?

Reply:
gotype was catching a false positive in a variable not being defined in the sources package, which led to roughly two hours of serious golang soul searching. The code works correctly, it is a false positive as far as I can tell.